### DISCUSSION TITLE
Injector coding

### DISCUSSION THREAD

**Discussion by didic-59** [cedricp/ddt4all#1059] [Q&A]
Hello,

I would like to change my 4 injectors with new ones on my Scenic 2 1.9 Dci 130 hp with 300,000 km
I cannot find the page in the DDT4ALL software to enter the flow codes written on the injectors. Do you have a tutorial or walkthrough please?

Note: this is a BOSCH type injector reference 0445110280

Thank you in advance for your help.
Best regards.

**Comment by didic-59** [comment]
Hello,

My post is not active ;-)
Has no one done any coding on DDT4ALL or is it because it's not possible?

Best regards

**Comment by KarelSvo** [comment]
email?

**Comment by didic-59** [comment]
Hello KarelSvo, 

Why you want my e-mail plz ? 

Best regards
